.github/workflows/idp-login-request-generator.yml

```diff
@@ -34,13 +34,16 @@ jobs:
           - runner-id: 6
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
       - id: 'auth'
-        uses: 'google-github-actions/auth@v0'
+        name: 'Authenticate to Google Cloud'
+        uses: 'google-github-actions/auth@v1'
```

Comment:
any reason not to pick the latest version here, v2.1.0 (https://github.com/google-github-actions/auth/releases)?

Reply:
thanks Justin, no, I just used a version I knew would work from a prior ticket.
does `@v2` sound good?